### ISSUE TITLE
Enhancements

### ISSUE THREAD

**Issue by lucas2107** [JavaFactoryDev/PunishControl-Wiki#1]
Hello,

Your plugin is looking very good, but there's some features missing.

Like AdvancedBans layout system, automatically set the duration of a ban, based on the used layout. Maybe, a bonus feature from Advancedban, add a point system to it, so, someone who isn't  sanctioned for a long time, don't get a very high ban/mute time.

Like others bans/mute systems, add a MySQL support, so the community can make a banlist or share bans with an other bungee instance.

**Comment by JavaFactoryDev** [comment]
Thank you for using PunishControl!
Yes, the journey has just started, I think I will implement features like that in the future:)

**Comment by lucas2107** [comment]
Your plugin is interesting, i don't know how much time you need to add that, but I hope you're fast x)

**Comment by JavaFactoryDev** [comment]
Yeah, I really think it has to potential to revolutionize somethings. Also, it is the first GUI-BungeeCord Plugin ever released. And I've many more things in my mind:)

**Comment by JavaFactoryDev** [comment]
Just a quick update: We are about completing the basic MySQL data structure in PunishControl.
